tests/data/snake_oil_data.py

```diff
@@ -3,6 +3,11 @@
 import pandas as pd
 
 
+ENSEMBLE_NAME_3 = "default3"
```

Comment:
really not sure if i like these name refactorings in the actual PR. For me they are messing up with raw test data as json, but i'm open for other opinions of course. To begin with this was dumping text from the endpoints, and i think it should be possible to to this in the future as well if needed... :man_shrugging: